### DISCUSSION TITLE
My achievements are not showing

### DISCUSSION THREAD

**Discussion by SureshChoudhary8284** [community/community#86567] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

In my profile, achievements are not showing. Someone help me?

**Comment by PrashantRatanparikhe** [comment, marked as answer]
Hey 
Have you checked the achievements on the profile page? If not, go to settings, verify it, and then update the preference setting.

Clear browser cahce and check again
![image_2024_01_11T08_04_57_038Z](https://github.com/community/community/assets/11390157/0fc68862-c5bd-4bcd-9495-42b73f246a41)

Thanks

**Reply by hurrainjhl** [reply]
i have tried every thing but it is not working
